vendor/node-usb-native/lib/serialport.js

```diff
@@ -226,6 +226,20 @@ SerialPort.prototype.write = function(buffer, callback) {
     buffer = Buffer.from(buffer);
   }
 
+  switch (ending) {
+    case 'Newline':
+      buffer = Buffer.concat([buffer, Buffer.from('\n')]);
+      break;
+    case 'Carriage return':
+      buffer = Buffer.concat([buffer, Buffer.from('\r')]);
+      break;
+    case 'Both NL & CR':
```

Comment:
You’ve already defined a constant value for ‘Both NL & CR’, suggest using Enum with string value here since there’re 3 cases here.

Reply:
Fixed.